<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elevator Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVBoxLayout" name="verticalLayout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="Testing">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Testing Section</string>
    </property>
    <widget class="QPushButton" name="buttonObstacle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Door Obstacle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOverload">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>128</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload Scenario</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonPowerOutage">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Outage</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFire">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>92</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire Scenario</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonHelp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>184</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(237, 51, 59);
background-color: rgb(224, 27, 36);</string>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CallElevator">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>120</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Call Elevator</string>
    </property>
    <widget class="QPushButton" name="buttonUp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDown">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QComboBox" name="startingfloor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>120</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Floors</string>
    </property>
    <widget class="QPushButton" name="buttonFloor1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFloor2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFloor3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor 3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFloor4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor 4</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="currfloordisplay">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Elevator1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Floor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Elevator 1:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Elevator2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Elevator 2:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVBoxLayout</class>
   <extends>QWidget</extends>
   <header>qvboxlayout.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
